app/assets/javascripts/3dbio_viewer/src/webapp/components/molecular-structure/usePdbPlugin.ts

```diff
@@ -77,7 +78,9 @@ export function usePdbePlugin(options: MolecularStructureProps) {
     const pdbePlugin = pdbePlugin0 && pluginLoad ? pdbePlugin0 : undefined;
     const chainId = newSelection.chainId;
     const ligandId = newSelection.ligandId;
-    const chains = React.useMemo(() => options.pdbInfo?.chains ?? [], [options.pdbInfo?.chains]);
+
+    const chains = React.useMemo(() => pdbInfo?.chains ?? [], [pdbInfo?.chains]);
```

Comment:
Not important, but here you are using useMemo as a way to keep the value immutable when it takes the fallback empty array, right? In that case, the typical approach is: `const chains = pdbInfo?.changes || emptyArray`. Where `const emptyArray = []` is defined at the root level, so it's immutable.